diff --git a/sonic-screenlocker/sonic-screenlocker.spec b/sonic-screenlocker/sonic-screenlocker.spec
--- a/sonic-screenlocker/sonic-screenlocker.spec
+++ b/sonic-screenlocker/sonic-screenlocker.spec
@@ -48,1 +48,1 @@
-# BuildRequires:	cmake(KF6GlobalAccel)
+BuildRequires: cmake(KF6GlobalAccel)
@@ -61,1 +61,9 @@
-#BuildRequires:	cmake(KF6KIO)
+BuildRequires: cmake(KF6KIO)
+BuildRequires: cmake(KF6WindowSystem)
+BuildRequires: cmake(KF6XmlGui)
+BuildRequires: cmake(KF6Bookmarks)
+BuildRequires: cmake(KF6Completion)
+BuildRequires: cmake(KF6ItemViews)
+BuildRequires: cmake(KF6JobWidgets)
+BuildRequires: cmake(KF6Solid)
+BuildRequires: cmake(KF6KirigamiPlatform)
@@ -77,1 +77,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -119,1 +119,2 @@
+Provides:       %{_lib}kscreenlocker-devel
 Conflicts:      %{_lib}kscreenlocker-devel
@@ -130,2 +130,2 @@
-# %{_libdir}/cmake/KScreenLocker
-# %{_libdir}/cmake/ScreenSaverDBusInterface
+%{_libdir}/cmake/KScreenLocker
+%{_libdir}/cmake/ScreenSaverDBusInterface
