diff --git a/sonic-frameworks-windowsystem/sonic-frameworks-windowsystem.spec b/sonic-frameworks-windowsystem/sonic-frameworks-windowsystem.spec
--- a/sonic-frameworks-windowsystem/sonic-frameworks-windowsystem.spec
+++ b/sonic-frameworks-windowsystem/sonic-frameworks-windowsystem.spec
@@ -46 +46,0 @@
-Requires: sonic-win
@@ -47,1 +47,3 @@
+Provides: cmake(KF6WindowSystem)
+Provides: pkgconfig(KF6WindowSystem)
 Conflicts: kf6-kwindowsystem
@@ -74,2 +74,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
-rm -rf %{buildroot}/%{_libdir}/pkgconfig
@@ -85,1 +85,2 @@
 %{_includedir}/KF6/KWindowSystem
+%{_qtdir}/metatypes/qt6kf6windowsystem_metatypes.json
@@ -88,2 +88,2 @@
-# %{_libdir}/cmake/KF6WindowSystem
-# %{_libdir}/pkgconfig/KF6WindowSystem.pc
+%{_libdir}/cmake/KF6WindowSystem
+%{_libdir}/pkgconfig/KF6WindowSystem.pc
