diff --git a/sonic-frameworks-auth/sonic-frameworks-auth.spec b/sonic-frameworks-auth/sonic-frameworks-auth.spec
--- a/sonic-frameworks-auth/sonic-frameworks-auth.spec
+++ b/sonic-frameworks-auth/sonic-frameworks-auth.spec
@@ -52,1 +52,2 @@
+Provides:      kf6-kauth
 Conflicts:     kf6-kauth
@@ -61,1 +61,2 @@
+Provides: %{_lib}KF6Auth
 Conflicts:  %{_lib}KF6Auth
@@ -70,1 +70,2 @@
+Provides: %{_lib}KF6Auth-devel
 Conflicts:  %{_lib}KF6Auth-devel
@@ -79,1 +79,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -92,1 +92,1 @@
-# %{_libdir}/cmake/KF6Auth
+%{_libdir}/cmake/KF6Auth
