diff --git a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
--- a/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
+++ b/sonic-frameworks-core-addons/sonic-frameworks-core-addons.spec
@@ -52,1 +52,2 @@
+Provides: kf6-kcoreaddons
 Conflicts:     kf6-kcoreaddons
@@ -65,1 +65,2 @@
+Provides: %{_lib}KF6CoreAddons
 Conflicts: %{_lib}KF6CoreAddons
@@ -74,1 +74,2 @@
+Provides: %{_lib}KF6CoreAddons-devel
 Conflicts: %{_lib}KF6CoreAddons-devel
@@ -85,1 +85,2 @@
+Provides: python-kcoreaddons
 Conflicts: python-kcoreaddons
@@ -91,1 +91,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -100,1 +100,2 @@
+%{_qtdir}/metatypes/qt6kf6coreaddons_metatypes.json
 %{_includedir}/KF6/KCoreAddons
@@ -103,1 +103,1 @@
-# %{_libdir}/cmake/KF6CoreAddons
+%{_libdir}/cmake/KF6CoreAddons
