diff --git a/sonic-frameworks-keybind/sonic-frameworks-keybind.spec b/sonic-frameworks-keybind/sonic-frameworks-keybind.spec
--- a/sonic-frameworks-keybind/sonic-frameworks-keybind.spec
+++ b/sonic-frameworks-keybind/sonic-frameworks-keybind.spec
@@ -46,1 +46,2 @@
+Provides:      kf6-kglobalaccel
 Conflicts:     kf6-kglobalaccel
@@ -55,1 +55,2 @@
+Provides: %{_lib}KF6GlobalAccel
 Conflicts: %{_lib}KF6GlobalAccel
@@ -64,1 +64,3 @@
+Provides: cmake(KF6GlobalAccel)
+Provides: %{_lib}KF6GlobalAccel-devel
 Conflicts: %{_lib}KF6GlobalAccel-devel
@@ -71,1 +71,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -81,1 +81,1 @@
-# %{_libdir}/cmake/KF6GlobalAccel
+%{_libdir}/cmake/KF6GlobalAccel
