diff --git a/sonic-audio-applet-pulse/sonic-audio-applet-pulse.spec b/sonic-audio-applet-pulse/sonic-audio-applet-pulse.spec
--- a/sonic-audio-applet-pulse/sonic-audio-applet-pulse.spec
+++ b/sonic-audio-applet-pulse/sonic-audio-applet-pulse.spec
@@ -26,1 +26,2 @@
-# BuildRequires: cmake(Plasma) >= 5.90.0
+BuildRequires: cmake(Plasma)
+BuildRequires: cmake(KF6WindowSystem)
@@ -69,1 +69,2 @@
 %{_qtdir}/plugins/plasma/applets/org.kde.plasma.volume.so
+%doc %{_docdir}/HTML/*
