diff --git a/sonic-screen-library/sonic-screen-library.spec b/sonic-screen-library/sonic-screen-library.spec
--- a/sonic-screen-library/sonic-screen-library.spec
+++ b/sonic-screen-library/sonic-screen-library.spec
@@ -49,1 +49,2 @@
+Provides: %{_lib}KF6Screen
 Conflicts:  %{_lib}KF6Screen
@@ -80,1 +80,4 @@
+Provides: cmake(KF6Screen)
+Provides: pkgconfig(KF6Screen)
+Provides: %{_lib}KF6Screen-devel
 Conflicts:      %{_lib}KF6Screen-devel
@@ -64,2 +64,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
-rm -rf %{buildroot}/%{_libdir}/pkgconfig
@@ -90,2 +90,2 @@
-# %{_libdir}/cmake/KF6Screen
-# %{_libdir}/pkgconfig/*.pc
+%{_libdir}/cmake/KF6Screen
+%{_libdir}/pkgconfig/*.pc
