diff --git a/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec b/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
--- a/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
+++ b/sonic-frameworks-quick-ui/sonic-frameworks-quick-ui.spec
@@ -47,1 +47,2 @@
+Provides:    kf6-kirigami
 Conflicts:   kf6-kirigami
@@ -60,1 +60,2 @@
+Provides: %{_lib}KirigamiPlatform
 Conflicts: %{_lib}KirigamiPlatform
@@ -69,1 +69,2 @@
+Provides: %{_lib}KirigamiPlatform-devel
 Conflicts: %{_lib}KirigamiPlatform-devel
@@ -75,1 +75,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -82,1 +82,2 @@
+%{_qtdir}/metatypes/qt6kirigamiplatform_metatypes.json
 %{_includedir}/KF6/Kirigami
@@ -85,1 +85,1 @@
-# %{_libdir}/cmake/KF6Kirigami*
+%{_libdir}/cmake/KF6Kirigami*
