diff --git a/sonic-interface-libraries/sonic-interface-libraries.spec b/sonic-interface-libraries/sonic-interface-libraries.spec
--- a/sonic-interface-libraries/sonic-interface-libraries.spec
+++ b/sonic-interface-libraries/sonic-interface-libraries.spec
@@ -47,1 +47,1 @@
-BuildRequires: %{_lib}SonicFrameworksQuickUI-devel
+BuildRequires: cmake(KF6KirigamiPlatform)
@@ -63,1 +63,1 @@
-BuildRequires: %{_lib}SonicDEKeybindDaemon-devel
+BuildRequires: cmake(KF6GlobalAccel)
@@ -72,1 +72,6 @@
+BuildRequires: cmake(KF6Bookmarks)
+BuildRequires: cmake(KF6Completion)
+BuildRequires: cmake(KF6ItemViews)
+BuildRequires: cmake(KF6JobWidgets)
+BuildRequires: cmake(KF6Solid)
 BuildRequires: %{_lib}SonicFrameworksIO-devel
@@ -97,1 +97,2 @@
+Provides: %{_lib}Plasma
 Conflicts:       %{_lib}Plasma
@@ -106,1 +106,4 @@
+Provides: cmake(Plasma)
+Provides: cmake(PlasmaQuick)
+Provides: %{_lib}Plasma-devel
 Conflicts:  %{_lib}Plasma-devel
@@ -132,1 +132,0 @@
-rm -rf %{buildroot}/%{_libdir}/cmake
@@ -143,2 +143,2 @@
-# %{_libdir}/cmake/Plasma
-# %{_libdir}/cmake/PlasmaQuick
+%{_libdir}/cmake/Plasma
+%{_libdir}/cmake/PlasmaQuick
