diff --git a/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec b/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
--- a/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
+++ b/sonic-frameworks-open-collab/sonic-frameworks-open-collab.spec
@@ -30 +30 @@
-BuildRequires: cmake(EGL)
+BuildRequires: mesa-libEGL-devel
@@ -40 +40 @@
-BuildRequires: qt6-qtbase-sql-sqlite
+BuildRequires: qt6-qtbase

