diff --git a/sonic-keybind-daemon/sonic-keybind-daemon.spec b/sonic-keybind-daemon/sonic-keybind-daemon.spec
--- a/sonic-keybind-daemon/sonic-keybind-daemon.spec
+++ b/sonic-keybind-daemon/sonic-keybind-daemon.spec
@@ -58,1 +58,7 @@
+BuildRequires: cmake(KF6DBusAddons)
+BuildRequires: cmake(KF6Bookmarks)
+BuildRequires: cmake(KF6Completion)
+BuildRequires: cmake(KF6ItemViews)
+BuildRequires: cmake(KF6JobWidgets)
+BuildRequires: cmake(KF6Solid)
 BuildRequires: pkgconfig(xkbcommon)
